docs/DeveloperGuide.md

```diff
@@ -461,32 +566,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given student is invalid/cannot be found.
+* 3a. The inputted student's name is invalid or cannot be found.
 
     * 3a1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
+
+***
+
+**Use case: Add a task for a student**
 
-* 3b. Multiple school level subjects were given.
+**MSS**
+
+1.  Tuition teacher requests to list students.
+2.  EduManage shows a list of students.
+3.  Tuition teacher requests to add a task to a specific student in the list by inputting the student's name, task description and task deadline.
+4.  EduManage adds the task to the student’s outstanding task list and indicates success.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The inputted student's name is invalid or cannot be found.
+
+    * 3a1. EduManage shows an error message.
+
+      Use case resumes at step 3.
+
+* 3b. The inputted task deadline is invalid.
 
     * 3b1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
 
-* 3c. No subjects were given.
+* 3c. No task description was inputted.
 
     * 3c1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
+
+***
 
-**Use case: View specific student**
+**Use case: Delete a task for a student**
 
 **MSS**
 
-1.  Tuition teacher requests to list students.
-2.  EduManage shows a list of students.
-3.  Tuition teacher requests to view a specific student.
-4.  EduManage shows the details from the student.
+1.  Tuition teacher requests to view all tasks for all students.
+2.  EduManage shows a list of students and their tasks.
+3.  Tuition teacher requests to delete a task from a specific student in the list by inputting the student's name and task index.
```

Comment:
Tuition teacher requests to delete a task from a specific student's task list by inputting the student's name and task index.